### ISSUE TITLE
Crash in Foundry 0.8.6

### ISSUE THREAD

**Issue by benbarbour** [VanirDev/VariantEncumbrance#36]
Editing my inventory causes a console error:

```
ReferenceError: SetupConfiguration is not defined
    convertItemSet VariantEncumbrance.js:189
    calculateEncumbrance VariantEncumbrance.js:358
    <anonymous> VariantEncumbrance.js:58
    _call lib-wrapper.js line 1 > eval:4
    call foundry.js:179
    _render foundry.js:2096
    call_wrapped libWrapper-wrapper.js:448
    <anonymous> libWrapper-errors.js:268
    0 libWrapper-wrapper.js:142
    _render foundry.js:2728
    render foundry.js:2003
    render foundry.js:3221
    render foundry.js:9346
    _onUpdateEmbeddedDocuments foundry.js:9473
    _onUpdateEmbeddedDocuments foundry.js:12346
    _handleUpdateEmbeddedDocuments foundry.js:8839
    _updateEmbeddedDocuments foundry.js:8725
    update backend.mjs:153
    updateDocuments document.mjs:365
    update document.mjs:447
    callOriginalFunction Monkey.js:80
    patchCompendiumUpdateEntity patches.js:100
    activateListeners tidy5e-sheet.js:154
    jQuery 9
    activateListeners tidy5e-sheet.js:148
    _render foundry.js:2069
    call_wrapped libWrapper-wrapper.js:448
    <anonymous> libWrapper-errors.js:268
    0 libWrapper-wrapper.js:142
    _render foundry.js:2728
    render foundry.js:2003
    render foundry.js:3221
    render foundry.js:9346
    _onUpdateEmbeddedDocuments foundry.js:9473
    _onUpdateEmbeddedDocuments foundry.js:12346
    _handleUpdateEmbeddedDocuments foundry.js:8839
    _updateEmbeddedDocuments foundry.js:8725
    update backend.mjs:153
    updateDocuments document.mjs:365
    update document.mjs:447
lib-wrapper.js line 1 > eval:8:15
    _call lib-wrapper.js line 1 > eval:8
    call foundry.js:179
    _render foundry.js:2096
    call_wrapped libWrapper-wrapper.js:448
    <anonymous> libWrapper-errors.js:268
    0 libWrapper-wrapper.js:142
    _render foundry.js:2728
    render foundry.js:2003
    render foundry.js:3221
    render foundry.js:9346
    _onUpdateEmbeddedDocuments foundry.js:9473
    _onUpdateEmbeddedDocuments foundry.js:12346
    _handleUpdateEmbeddedDocuments foundry.js:8839
    _updateEmbeddedDocuments foundry.js:8725
    update backend.mjs:153
    updateDocuments document.mjs:365
    update document.mjs:447
    callOriginalFunction Monkey.js:80
    patchCompendiumUpdateEntity patches.js:100
    activateListeners tidy5e-sheet.js:154
    jQuery 9
    activateListeners tidy5e-sheet.js:148
    _render foundry.js:2069
    call_wrapped libWrapper-wrapper.js:448
    <anonymous> libWrapper-errors.js:268
    0 libWrapper-wrapper.js:142
    _render foundry.js:2728
    render foundry.js:2003
    render foundry.js:3221
    render foundry.js:9346
    _onUpdateEmbeddedDocuments foundry.js:9473
    _onUpdateEmbeddedDocuments foundry.js:12346
    _handleUpdateEmbeddedDocuments foundry.js:8839
    _updateEmbeddedDocuments foundry.js:8725
    update backend.mjs:153
    updateDocuments document.mjs:365
    update document.mjs:447

```

**Comment by dsakura** [comment]
Not working on 0.8.6. Hope you can update. Thank you

**Comment by naesean** [comment]
Is this project being maintained? I updated to 0.8.6 and now I can't get to anything in the PC's backpacks and other containers.

**Comment by VanirDev** [comment]
Working on 0.8.6 Compatability, should hopefully be resolved today or tomorrow.

**Comment by Drakynn** [comment]
I'm a bit confused on the versioning.   This issue was just added to the 0.3.1 milestone, but the current version is 0.3.2.

**Comment by ShroomDruid** [comment]
> 
> 
> Working on 0.8.6 Compatability, should hopefully be resolved today or tomorrow.

Thank you! Super useful module and there's no other that handles this.

**Comment by RavenknightDM** [comment]
This module is a must have. :) Hope to see 0.8+ soon.

**Comment by ZeeShinnie** [comment]
Hallo, do we have any updates on 0.8.x compatibility? I guess it is just that different...

**Comment by ShroomDruid** [comment]
> 
> 
> Working on 0.8.6 Compatability, should hopefully be resolved today or tomorrow.

Can you confirm whether or not you still plan on updating this module for the new version of Foundry? This is a super useful module..

**Comment by MattiaPandolfoVR** [comment]
Hi, any news? Thanks for your work, this module is a must have!

**Comment by p4535992** [comment]
Should be fixed with 0.4.6
